actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -29,8 +29,7 @@ class InvalidCrossOriginRequest < ActionControllerError #:nodoc:
   # you're building an API you'll need something like:
   #
   #   class ApplicationController < ActionController::Base
-  #     protect_from_forgery
-  #     skip_before_action :verify_authenticity_token, if: :json_request?
+  #     protect_from_forgery unless: json_request?
```

Comment:
And while we're at it, let's inline it: `protect_from_forgery unless: -> { request.format.json? }`